application/models/Template.php

```diff
@@ -293,6 +293,10 @@ public static function getTemplatePath($sTemplateName = "")
         }
 
         $oTemplate = self::model()->findByPk($sTemplateName);
+        if (empty($oTemplate))
```

Comment:
https://www.php-fig.org/psr/psr-12/

Should be

```php
if (empty($oTemplate)) {
```

`{` on the same line. You should include PSR-12 checking in your editor (will show LOTS of errors, but make sure all new code follows it, please).

Reply:
Fixed